guides/k8-akka-cluster/src/main/paradox/index.md

```diff
@@ -0,0 +1,455 @@
+
+Append the Akka clustering related sytem properties to the `dockerEntrypoint` setting.
+
+```
+dockerEntrypoint ++= """-Dplay.crypto.secret="${APPLICATION_SECRET:-none}" -Dplay.akka.actor-system="${AKKA_ACTOR_SYSTEM_NAME:-friendservice-v1}" -Dhttp.address="$FRIENDSERVICE_BIND_IP" -Dhttp.port="$FRIENDSERVICE_BIND_PORT" -Dakka.actor.provider=cluster -Dakka.remote.netty.tcp.hostname="$AKKA_REMOTING_BIND_HOST" -Dakka.remote.netty.tcp.port="$AKKA_REMOTING_BIND_PORT" -Dakka.cluster.seed-nodes.0="akka.tcp://${AKKA_ACTOR_SYSTEM_NAME}@${AKKA_SEED_NODE_HOST}:${AKKA_SEED_NODE_PORT}" -Dakka.io.dns.resolver=async-dns -Dakka.io.dns.async-dns.resolve-srv=true -Dakka.io.dns.async-dns.resolv-conf=on""".split(" ").toSeq
+```
+
+As part of building the Docker image, SBT Native Packager will provide its own Docker entry point script to start the application. which accepts optional additional arguments. When system properties are presented as part of the arguments, they will be appended to the JVM options when the application is started within the container.
+
+Next we will transform the generated Docker `ENTRYPOINT` instruction.
+
+```
+dockerCommands :=
+  dockerCommands.value.flatMap {
+    case ExecCmd("ENTRYPOINT", args @ _*) => Seq(Cmd("ENTRYPOINT", args.mkString(" ")))
+    case v => Seq(v)
+  }
+```
+
+The script above transforms `ENTRYPOINT` from `exec` form to `shell` form. The `shell` form is required to ensure the environment variables declared in the `dockerEntrypoint` argument is sourced from the shell within the Docker container. Refer to Docker [ENTRYPOINT](https://docs.docker.com/engine/reference/builder/#entrypoint) documentation for the difference between `exec` vs `shell` form.
+
+When the image is published from the build, the image will be published to the default Docker repository. To publish to a different repository, set the `dockerRepository` setting to the repository you wish to publish. Please note this is an optional step.  The Kubernetes deployment in this guide expects the image to be located at `mygroup/myapp`. Assuming the SBT project name is `myapp`, to match this Docker image location, to match this expectation we need to set the `dockerRepository` as following.
+
+```
+dockerRepository := Some("mygroup")
+```
+
+When the image is published from the build, the version tag will be derived from the SBT project version. However it's possible to also publish both the `latest` tag and the version tag by enabling the `dockerUpdateLatest` setting. Please note this is an optional step. The Kubernetes deployment in this guide expects the `latest` image tag to be present. To match this expectation we need to enable `dockerUpdateLatest` setting.
+
+```
+dockerUpdateLatest := true
+```
+
+For various SBT settings available to control the Docker image build process, refer to the [Docker Plugin](http://www.scala-sbt.org/sbt-native-packager/formats/docker.html?highlight=dockercommand) documentation page from SBT Native Packager.
```

Comment:
Change to: `Additional SBT setting documentation to control the Docker image build process is available at the [Docker Plugin](http://www.scala-sbt.org/sbt-native-packager/formats/docker.html?highlight=dockercommand) documentation page from SBT Native Packager.`